docs/developer-guide/processing-log.rst

```diff
@@ -158,6 +158,8 @@ The replication factor and partition count are configurable
 using the ``ksql.processing.log.topic.replication.factor`` and ``ksql.processing.log.topic.partitions`` properties,
 respectively.
 
+If ``ksql.processing.log.topic.auto.create`` is set to ``true``, the created topic will be deleted as part of :ref:`cluster termination<ksql_cluster_terminate>`.
```

Comment:
Would this mean that we lose the logs for the cluster? How would we diagnose a cluster that was accidentally terminated?

Reply:
The processing logs, yes. Not the operational log, which is what we'd use to debug the type of issue you've described. The processing log lets users who are writing/managing ksql queries introspect what happens as ksql is processing records. So, for example, the processing log would tell a user about a deserialization failure when trying to read a row, or about the inputs/outputs to an aggregation (in debug mode). The operational log would be the place an operator would look to investigate problems running the ksql service itself (e.g. an error reading from the command topic).